scripts/india_post/pincodes/IndiaPost_Pincodes.mcf

```diff
@@ -1,5 +1,12 @@
-Node: dcid:PincodeArea
+Node: dcid:PinCodeArea
 typeOf: schema:Class
 subClassOf: schema:AdministrativeArea
-name: "PincodeArea"
-description: "Postal delivery area representations of India Post."
\ No newline at end of file
+name: "PinCodeArea"
+description: "Postal delivery area representations of India Post."
+
+Node: dcid:pinCode
+typeOf: schema:Property
+name: "pinCode"
+description: "PIN code is a six digit code in Indian postal code system used by India Post."
+domainIncludes: schema:Place
+rangeIncludes: schema:Number
```

Comment:
Can it have 0 prefix? If so we can make it Text

Reply:
Currently no.

Its 1-9. It indicates the zone.


But we can make it Text just to be safe.